### PR TITLE
Re-render the tooltip element when the content prop changes

### PR DESCRIPTION
**Pull request by Chochek**

### CODE REVIEW

**Comment by Chochek** [comment]
[(1) Reactions: Re-render the tooltip element when the content prop changes](https://trello.com/c/PR5usZ34/706-1-reactions-re-render-the-tooltip-element-when-the-content-prop-changes)

File: src/components/Tooltip/Tooltip.js [hunk 1]
```diff
@@ -74,17 +74,13 @@ class Tooltip extends React.Component {
   }
 
   componentWillReceiveProps = (nextProps) => {
-    // If not currently showing and should show
-    if (!this.state.showing && nextProps.show) {
-      this.setState({ showing: true }, () => {
-        this.showTip()
-      })
+    // If not currently showing and should show or already showing but the content changed
+    if (!this.state.showing && nextProps.show || nextProps.show && nextProps.content !== this.props.content) {
```
Comment (gpszymczak) [inline src/components/Tooltip/Tooltip.js]: @Chochek you could probably avoid repeating this `nextProps.show || nextProps.show`?

Comment (fieldsco) [inline src/components/Tooltip/Tooltip.js]: @gpszymczak @Chochek I think the repeat has to stay, but this needs innner parentheses after the ||

Comment (gpszymczak) [inline src/components/Tooltip/Tooltip.js]: it can be if a && (b || c)

Comment (gpszymczak) [inline src/components/Tooltip/Tooltip.js]: so it will be `nextProps.show && (!this.state.showing || nextProps.content !== this.props.content)`

**Pull request merged**